index.html

```diff
@@ -0,0 +1,302 @@
+                                <li class="card-tag css"><p class="card-tag-text">css</p></li>
+                                <li class="card-tag Javascript"><p class="card-tag-text">Javascript</p></li>
+                                <li class="card-tag html"><p class="card-tag-text">html</p></li>
+                            </ul>
+
+                            <div class="card-title-button">
+                                <button class="card-button"><p class="card-button-text">See Project</p></button>
+                            </div>
+                        </section>
+                    </div>
+
+                    <!-- card3 -->
+                    <div class="card">
+                        <section class="card-content">
+                            <h4 class="card-title">Multi-Post Stories Gain+Glory</h4>
+                            
+                            <ul class="card-tags">
+                                <li class="card-tag ruby"><p class="card-tag-text">Ruby on rails</p></li>
+                                <li class="card-tag css"><p class="card-tag-text">css</p></li>
+                                <li class="card-tag Javascript"><p class="card-tag-text">Javascript</p></li>
+                                <li class="card-tag html"><p class="card-tag-text">html</p></li>
+                            </ul>
+
+                            <div class="card-title-button">
+                                <button class="card-button"><p class="card-button-text">See Project</p></button>
+                            </div>
+                        </section>
+                    </div>
+                </section>
+            </div>
+
+            <div>
+
+            </div>
+        </section>
+        
+        <!-- About me Section -->
+        <section id="about">
+            <div id="about-heading">    
+                <h3 id="about-title">About me</h3>
+
+                <p class="about-description description-text"> I can help you build a product , 
+                    feature or website Look through 
+                    some of my work and experience! 
+                    If you like what you see and have 
+                    a project you need coded, 
+                    don't hestiate to contact me.</p>
+                <a href="resume.pdf" download="My Resume"><button id="about-resume-button"><p id="about-resume-button-text">Get My Resume</p></button></a>
+            </div>
+            
+            <div id="skill-cards">
+                <!-- languages -->
+                <section class="skill-card">
+                    <!-- icon -->
+                    <div class="skill-card-icon">
+                        <div class="skill-card-icon-container"><img class="skill-card-icon-img" src="img/skill/language-img.svg" alt=""></div>
+                    </div>
+
+                    <!-- title and tags -->
+                    <div class="skill-card-content">
```

Comment:
- [ ] Your skills cards content is overlapping due to the usage of `position: relative;` in `skill-card-icon` and `skill-card-content`. kindly consider removing it and then try to layout your card content with flex as suggested in the project requirements and  try to design it to make it similar to the figma design.
your design
![zx](https://user-images.githubusercontent.com/96683843/219133689-fc0744a4-3b72-4f9b-b731-01786120e87c.jpg)

figma design
 
![vc](https://user-images.githubusercontent.com/96683843/219133666-3eee87d8-72bb-4d0a-aa39-06842bfbc284.jpg)